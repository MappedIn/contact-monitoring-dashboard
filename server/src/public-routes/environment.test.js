process.env.npm_package_version = 'npm_package_version';

const supertest = require('supertest');
const express = require('express');
const environmentRoutes = require('./environment');

describe('/environment', () => {
	let app;
	beforeAll(() => {
		app = express();
		environmentRoutes(app);
	});

	test('returns 200', async () => {
		const response = await supertest(app)
			.get('/environment')
			.expect('Cache-Control', 'private, must-revalidate')
			.expect('Content-Type', 'application/javascript; charset=utf-8')
			.expect(200);

		expect(response.text).toMatchSnapshot();
	});
});
