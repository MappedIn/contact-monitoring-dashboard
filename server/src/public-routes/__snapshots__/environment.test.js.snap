// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/environment returns 200 1`] = `"window.MI = {\\"auth\\":{},\\"hosts\\":{},\\"version\\":\\"npm_package_version\\"};"`;
